lib/dom.js

```diff
@@ -395,6 +395,7 @@ declare class Element extends Node {
     scrollWidth: number;
     tagName: string;
 
+    closest(): void;
```

Comment:
`closest()` isn't a standard DOM method. It's experimental, but I don't think that's enough to add to the core library definition.